src/js/components/Video.js

```diff
@@ -129,6 +129,10 @@ export default class Video extends Component {
   }
 
   _seek(time) {
+    if (time === 0) {
+      return this._video.currentTime = time;
+    }
+
     this._video.currentTime = time || this._video.currentTime;
```

Comment:
I think we can simply this to:

```
this._video.currentTime = typeof time !== 'undefined' ? time : this._video.currentTime;
```

As long as we are not using `time = null` we should be good